index.html

```diff
@@ -185,5 +185,37 @@ <h2>
         </div>
     </section>
 
+    <section class="contact-me">
+        <div class="form">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/mlezzzeo" method="post" class="form">
+                <input id="name" name="name" type="text" placeholder="Full Name" required="required" maxlength="10">
```

Comment:
- [ ] the max length for the name input should be 30 according to the requirements.

Reply:
Slight typo error. Fixed.